content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md

```diff
@@ -0,0 +1,263 @@
+# Importing External Files using Webpack Loaders: The Beginners Guide 
+Today, I am going to show you how to use Webpack loaders and import external files while making use of Webpack. In this tutorial, I am going to be very concise, because I have understood that making use of Webpack loaders to import external files in a web project, can be quite demanding and may look like rocket science, especially for beginners, but it is actually quite easy. 
+
+## Table Of Contents
+1. What is Webpack?
+2. Installing Webpack
+3. Loaders and Plugins
+4. Make use of Webpack loaders to import various types of files
+
+## Prerequisites
+This tutorial is tailored to suit the needs of a beginner, so if you are a beginner frontend developer who writes HTML, CSS, and normal JavaScript, then you are good to go. 
+
+**Let us dive in!**
+
+## What is Webpack?
+Webpack is a tool that allows you to modularise your code and helps you to push your codes to one file in other for you not to worry about the hierarchy of your files, especially when working with JavaScript files and also helps you to inject external files, like an SVG, images, your CSS files or other modern JS codes, in your web app. Webpack provides a configuration file, where you declare how things should be sorted out and then you relax. 
+
+**Remember I stated earlier that this tutorial is going to be concise, hence a very simple definition of Webpack, which is what I have stated above, for a more elaborate definition of Webpack, check out this Wiki Definition of Webpack [Wikipedia's Definition of Webpack](https://en.wikipedia.org/wiki/Webpack)**
+
+## Installing Webpack
+The first thing you are going to do is to create a directory or folder, you can call it Webpack or anything you feel appropriate, and then you create another folder and name it Webpack Tutorial. 
+
+<img src="./images/webpack1.png" alt="File directory image" border="0"></a>
```

Comment:
This is not markdown.

I would suggest you to have a look at the guidelines

Reply:
Fixed. Thank You